app/models/user.rb

```diff
@@ -321,6 +331,11 @@ def dashboard_row
     data_attributes.dup.merge(module_ttc)
   end
 
+  # @return [Training::Module]
```

Comment:
I think we should let the course object answer and not add an extra method on user. Is the doc accurate?

Reply:
[62482be](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/726/commits/62482befe8bddaab78d25d93bcc6384777f18953)